server/aws-lsp-codewhisperer/src/language-server/agenticChat/tools/executeBash.ts

```diff
@@ -315,7 +314,7 @@ export class ExecuteBash {
                 onStderr: async (chunk: string) => {
                     if (cancellationToken?.isCancellationRequested) {
                         this.logging.debug('Bash command execution cancelled during stderr processing')
-                        return
+                        throw new CancellationError('user')
```

Comment:
do we need to kill the process or is this sufficient?

Reply:
Just looked into this, it's already cancelled by https://github.com/aws/language-servers/blob/main/server/aws-lsp-codewhisperer/src/language-server/agenticChat/tools/executeBash.ts#L334